### DISCUSSION TITLE
Logo voting!

### DISCUSSION THREAD

**Discussion by theshmunguss1** [theshmunguss1/RamOS#6] [Announcements]
Logo voting has closed! [Voting link](https://forms.gle/A3nZ5VLLvBTC8Fsh9)
-
All designs will be placed into a Google Form and voted upon, and the winner will have their design set as the profile picture of the RamOS repository, and possibly the favicon on the website!

**There are some guidelines you must follow when making your logo:**
- No NSFW terms/images
- No fake PNG images
- Favicon sumbissions must have either a transparent background or rounded corners. It may also be a circle.
- For favicons, do not make them very detailed, you could limit yourself to a resolution around 300x300 pixels.
- For a profile picture, you may add more details.
- All submissions must represent something about RamWorks (either a pixel theme to games, film reel to movies, etc.).
- All submissions will be entered into the form, so do not post anything you do not want anyone to see.
- No spam. All comments must follow the correct format (see below), so keep feedback to replies.
- All submissions must be high-quality. Low-quality images will not be entered.
- The reccommended size of your images should be 200x200 pixels to not take up space in the comments. Your image source file may be larger than this resolution. Use the code below to resize your images.

`<img src="Image address goes here" width="200" height="200">`

**Comment by theshmunguss1** [comment]
Sample comment:

"favicon.png" is my favicon submission.
"profile.jpg" is my profile submission.

<img src="https://github.com/ramnetos/RamOS/assets/142841882/db0bc5fc-ae1f-4b40-8ac3-0dad30447ac6)" width="200" height="200">
<img src="https://github.com/ramnetos/RamOS/assets/142841882/35833b3e-f00d-4254-9676-58b3b661518b" width="200" height="200">

**Comment by YesTheProxy** [comment]
![Untitled](https://github.com/ramnetos/RamOS/assets/131057815/257da303-a9cc-469f-8500-4ff1f85217f0)
Favicon submition

**Comment by theshmunguss1** [comment]
Profile picture:
<img src="https://github.com/ramnetos/RamOS/assets/142841882/fb80c25c-064f-459c-a0a1-4c6246b1d693)" width="200" height="200">
Favicon:
<img src="https://github.com/ramnetos/RamOS/assets/142841882/d51d6cbe-5c6b-4067-9cb0-d7604838f895)" width="200" height="200">

**Comment by YesTheProxy** [comment]
![Untitled(1)](https://github.com/ramnetos/RamOS/assets/131057815/427aa123-df07-44a1-8bae-00df4fcfb4a7)
Profile (I took inspo from zach)

**Comment by YesTheProxy** [comment]
![Ram OS Warm](https://github.com/ramnetos/RamOS/assets/131057815/3057ec19-304f-461d-8a4c-a87be128fb1a)
And this is just a warm version of his favicon
